test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
```

Comment:
@beauby not the same thing.  They must be defined inside the test or else they will interact and cause crazy bugs.  We could only get away with it when there were all in one file.  Believe me, I've encountered this.  I'm guessing your concern is just the namespacing.